packages/database/src/modelClasses/SurveyResponse.js

```diff
@@ -7,6 +7,21 @@ import { MaterializedViewLogDatabaseModel } from '../analytics';
 import { DatabaseType } from '../DatabaseType';
 import { TYPES } from '../types';
 
+const EXCLUDED_USERS = [
```

Comment:
```suggestion
const USERS_EXCLUDED_FROM_LEADER_BOARD = [
```
Perhaps a little clearer?